modules/topicsFpdModule.js

```diff
@@ -1,13 +1,31 @@
-import {logError, logWarn, mergeDeep} from '../src/utils.js';
+import {logError, logWarn, mergeDeep, isEmpty, safeJSONParse} from '../src/utils.js';
 import {getRefererInfo} from '../src/refererDetection.js';
 import {submodule} from '../src/hook.js';
 import {GreedyPromise} from '../src/utils/promise.js';
+import {config} from '../src/config.js';
+import {getStorageManager} from '../src/storageManager.js';
+import {includes} from '../src/polyfill.js';
 
+export const storage = getStorageManager();
+export const topicStorageName = 'prebid:topics';
+export const lastUpdated = 'lastUpdated';
+
+const iframeLoadedURL = [];
 const TAXONOMIES = {
   // map from topic taxonomyVersion to IAB segment taxonomy
   '1': 600
 }
 
+const DEFAULT_EXPIRATION_DAYS = 21;
+
+const bidderIframeList = {
+  maxTopicCaller: 1,
+  bidders: [{
+    bidder: 'pubmatic',
+    iframeURL: 'https://pubmatic.com:8080/topics/fpd/topic.html' // dummy URL for NOW
```

Comment:
is the dummy data useful here? might be better to leave this empty and pass the test urls via config.